Mixpanel/Mixpanel.m

```diff
@@ -325,17 +319,11 @@ - (id)JSONSerializableObjectForObject:(id)obj
 
 - (NSString *)encodeAPIData:(NSArray *)array
 {
-    NSString *b64String = @"";
     NSData *data = [self JSONSerializeObject:array];
     if (data) {
-        b64String = [data mp_base64EncodedString];
-        b64String = CFBridgingRelease(CFURLCreateStringByAddingPercentEscapes(kCFAllocatorDefault,
-                                                                (__bridge CFStringRef)b64String,
-                                                                NULL,
-                                                                CFSTR("!*'();:@&=+$,/?%#[]"),
-                                                                kCFStringEncodingUTF8));
+        return MPURLEncode([data mp_base64EncodedString]);
```

Comment:
Have a look at: [data base64EncodedStringWithOptions:NSDataBase64Encoding64CharacterLineLength];
It was introduced in iOS7, and then we can get rid of the entire file NSData+MPBase64